libpod/runtime_img.go

```diff
@@ -288,6 +288,11 @@ func (r *Runtime) LoadImage(ctx context.Context, name, inputFile string, writer
 		if err == nil {
 			return getImageNames(newImages), nil
 		}
+
+		newImages, err = r.ImageRuntime().LoadAllImagesFromOCIArchive(ctx, inputFile, signaturePolicy, writer)
+		if err == nil {
+			return getImageNames(newImages), nil
+		}
```

Comment:
These “try N different formats, succeed on any one” functions have frequently caused user confusion.

- Would it make sense to have the user specify the format? I guess auto-detection is a desirable feature…
- The error reporting definitely needs improving. If the user submits a docker-archive format that is mostly (or completely) valid but the load fails, the docker-archive-related error _must_ be available to the user.
- (Extra thought, probably not for this PR: It might be interesting to have an “autodetect archive format” helper so that we can detect the format separately (or report a clear error that the format is unrecognized), and then just do one load attempt with a simple error reporting structure. Or, well, we could go even further and share a single “decompress and untar” operation across all formats — right now we would decompress the multiple gigabytes separately for each format.)
- … actually, the fact that we have to decompress the multiple-gigabyte archive in every format until we find one that works is a pretty big cost, and IMHO a fairly strong reason to at least _allow_ the user to specify the format, even we still wanted to support auto-detection.